lib/install/management/virtual_app.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
```

Comment:
👀  🤔

Reply:
Thanks license-bot!

Reply:
This is actually a revert, so its not a new file containing changes from 2017 :)

Reply:
coolio, I figured it wasn't wrong because I remember having it, but also wasn't sure why it seemed to be new.